docs/metrics.md

```diff
@@ -50,6 +50,8 @@ history.check.success                     | meter     | history archive status c
 history.publish.failure                   | meter     | published failed
 history.publish.success                   | meter     | published completed successfully
 history.publish.time                      | timer     | time to successfully publish history
+history.get.throughput                    | meter     | bytes per second of history archive retrieval
+history.get.retry                         | counter     | count of retries to get history
```

Comment:
Nit: spacing is off for `history.get.retry`. I'd also make this a little more clear, something like "count of history archive download retries"